frontend/websites/puconvocation-com/src/app/[locale]/(console)/console/attendees/page.tsx

```diff
@@ -10,15 +10,167 @@
  * treaties. Unauthorized copying or distribution of this software
  * is a violation of these laws and could result in severe penalties.
  */
+"use client";
 
-import { JSX } from "react";
+import { Fragment, JSX, useState } from "react";
+import { useQuery } from "@tanstack/react-query";
+import { Button } from "@components/ui";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@components/ui/card";
+import { AttendeeService } from "@services/index";
+import { StatusCode } from "@enums/StatusCode";
+import {
+  UsersIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from "@heroicons/react/24/solid";
+import { ProgressBar } from "@components/index";
+
+const attendeeService = new AttendeeService();
 
 export default function AttendeePage(): JSX.Element {
+  const [showAttendees, setShowAttendees] = useState(false);
+  const [page, setPage] = useState(0);
+  const [attendeeCount, setAttendeeCount] = useState(0);
+
+  const {
+    data: attendees,
+    isLoading: isAttendeeLoading,
+    isError: attendeeError,
+  } = useQuery({
+    queryKey: ["homeAttendeesList", page],
+    refetchOnWindowFocus: false,
+    queryFn: async () => {
+      const response = await attendeeService.getAllAttendees(page, 10);
+      if (
+        response.statusCode === StatusCode.SUCCESS &&
+        "payload" in response &&
+        typeof response.payload === "object"
+      ) {
+        setAttendeeCount(response.payload.totalCount);
```

Comment:
There is no such Property named `totalCount` in the  [`AttendeeWithPagination.tsx`](https://github.com/mihirpaldhikar/pu-convocation/blob/main/frontend/websites/puconvocation-com/src/dto/AttendeeWithPagination.ts)

Here, what we actually mean by total attendee is the total attendees that exists in the database. Not the total attendees in the current page.

You can obtain the total attendee count by creating method in  [`AttendeeController.tsx` (previously known as AttendeeService.tsx)](https://github.com/mihirpaldhikar/pu-convocation/blob/main/frontend/websites/puconvocation-com/src/controllers/AttendeeController.ts). 

For your reference, [here](https://github.com/mihirpaldhikar/pu-convocation/blob/1da36c122be2e5c797c859404ce40277d26357ed/backend/services/dynamics-service/src/main/kotlin/com/puconvocation/routes/AttendeeRoute.kt#L64) is the intended HTTP route you need to call from the frontend.